forge-gui/res/cardsfolder/upcoming/nanogene_conversion.txt

```diff
@@ -0,0 +1,7 @@
+Name:Nanogene Conversion
+ManaCost:3 U
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Choose target creature you control | SubAbility$ DBRepeatEach | StackDescription$ Each other creature {p:You} controls becomes a copy of {c:Targeted} until end of turn, except those creatures aren't legendary if {c:Targeted} is legendary. | SpellDescription$ Choose target creature you control. Each other creature becomes a copy of it until end of turn, except those creatures aren't legendary if the chosen creature is legendary.
```

Comment:
wrong SpellDescription

Reply:
You should be mindful of StackDescriptions too:
![image](https://github.com/Card-Forge/forge/assets/8506892/053245e5-42f2-4916-9582-3f45896389c7)